src/reporters/udp_sender.js

```diff
@@ -46,7 +47,7 @@ export default class UDPSender {
                 maxPacketSize: number = UDP_PACKET_MAX_LENGTH) {
         this._hostPort = hostPort;
         this._maxPacketSize = maxPacketSize;
-        this._maxSpanBytes = this._maxPacketSize - EMIT_SPAN_BATCH_OVERHEAD;
+        this._maxSpanBytes = this._maxPacketSize - constants.EMIT_SPAN_BATCH_OVERHEAD;
```

Comment:
this old logic is now completely wrong, because the overhead depends on the size of the Process thrift.